src/selfplay/tournament.cc

```diff
@@ -28,6 +28,7 @@ namespace lczero {
 
 namespace {
 const char* kShareTreesStr = "Share game trees for two players";
+const char* kReuseTreeeStr = "Reuse the node statistics between moves.";
```

Comment:
No dot in the end.
(It's not usual to have them in UCI, and --help adds dot by itself).

Reply:
Done.